spacy_transformers/layers/transformer_model.py

```diff
@@ -15,7 +15,7 @@
 
 
 def TransformerModel(
-    name: str, get_spans: Callable, tokenizer_config: dict
+    name: str, get_spans: Callable, tokenizer_config: dict, transformers_config: dict
```

Comment:
@svlandeg Does it help us if we default this parameter to an empty dict? It seems very reasonable to default this to empty, and it helps with the backwards compatibility, right?

Reply:
Yep, agreed!

Reply:
Added this as well while I was at it. One point is that remove transformer_tok2vec_v2 entirely (would require setting transformer_config as the last element and adding only default to it.)

I assumed this wasn't desirable and kept the two versions.